src/_pytest/config/__init__.py

```diff
@@ -1305,6 +1316,69 @@ def _getconftest_pathlist(
             values.append(relroot)
         return values
 
+    def _get_append_ini_values(self, name: str, ini_type) -> List[str]:
```

Comment:
In terms of reading order, it will be better if `_get_override_ini_value` appears right next to `_get_override_ini_value` as they are related.